client/src/message_queue.rs

```diff
@@ -85,6 +85,7 @@ impl MessageQueue {
     pub(crate) fn store_response(&mut self, response: SupportedMessage) {
         // Remove corresponding request handle from inflight queue, add to responses
         let request_handle = response.request_handle();
+        trace!("Received response {:?}", response);
```

Comment:
Remove this

Reply:
Done (will push an additional commit with all requested changes in a bit)